parser.py

```diff
@@ -589,18 +609,47 @@ def parse_direct_messages(data_folder, username, users, user_id_url_template, dm
           f"({num_written_messages} total messages) to {num_written_files} markdown files\n")
 
 
+def init_paths():
+    dir_archive             = '.'                                                         # used to be `input_folder`
+    dir_input_data          = os.path.join(dir_archive,       'data')                     # used to be `data_folder`
+    dir_input_media         = find_dir_input_media(dir_input_data)                        # used to be `archive_media_folder`
+    dir_output_media        = os.path.join(dir_archive,       'media')                    # used to be `output_media_folder_name`
+    file_output_html        = os.path.join(dir_archive,       'TweetArchive.html')        # used to be `output_html_filename`
+    file_output_following   = os.path.join(dir_archive,       'following.txt')            # used to be `output_following_filename`
+    file_output_followers   = os.path.join(dir_archive,       'followers.txt')            # used to be `output_followers_filename`
+    file_template_dm_output = os.path.join(dir_archive,       'DMs-Archive-{}.md')        # used to be `dm_output_filename_template`
+    file_account_js         = os.path.join(dir_input_data,    'account.js')               # used to be `account_js_filename`
+    file_download_log       = os.path.join(dir_output_media,  'download_log.txt')         # used to be `log_path`
+    file_tweet_icon         = os.path.join(dir_output_media,  'tweet.ico')                # used to be `tweet_icon_path`
+    files_input_tweets      = find_files_input_tweets(dir_input_data)                     # used to be `input_filenames`
+
```

Comment:
I think we can lose the `#used to be`. If anyone has merge problems we can refer back to the comment in this PR.